src/cli/manager.rs

```diff
@@ -51,6 +51,15 @@ pub fn mavlink_connection_string() -> Option<&'static str> {
     return MANAGER.as_ref().clap_matches.value_of("mavlink");
 }
 
+pub(crate) fn log_path() -> String {
```

Comment:
Wait, what is the difference between `pub fn` and `pub(crate)` ?

Reply:
lol, I didn't see that, pub(crate) is public only for the crate, useful for libraries I guess, it was generated by the vscode, I'll change it.